urdfenvs/sensors/pseudo_sensor.py

```diff
@@ -0,0 +1,65 @@
+import numpy as np
+import pybullet as p
+import gym
+import sys
+from urdfenvs.sensors.sensor import Sensor
+
+
+class PseudoSensor(Sensor):
+    """
+    the PseudoSensor class is a sensor sensing the exact position of every object. The PseudoSensor is thus
+    a full information sensor which in the real world can never exist. The PseudoSensor returns a dictionary
+    with the position of every object when the sense function is called.
+    """
+
+    def __init__(self, obstacles=[], goals=[]):
+        super().__init__("pseudoSensor")
+        self._obstacles = obstacles
+        self._observation = np.zeros(self.getOSpaceSize())
+        self._goals = goals
+
+    def getOSpaceSize(self):
+        size = 0
+        for obj_id in range(2, p.getNumBodies()):
```

Comment:
I do agree, what if for example there is no robot (pretty hard since this:
`` gym.make('pointRobotUrdf-vel-v0', dt=0.05, render=True)``
is how to make the environment, which includes the robot)
I shall check if gym stores any info which makes a distinction

Reply:
That would be great, maybe by the name of body. But if there is nothing, we don't need it here.